src/components/Feedback/Feedback.tsx

```diff
@@ -164,28 +165,9 @@ export const Feedback = ({
     if (!customBadge) { return }
 
     return (
-      <div className={styles.badge}>
-        <div className={styles.badgeIcon} data-testid="badge-icon">
-          <Icon color={palette.text.neutral.subtle} component={customBadge.icon} size={48} />
-        </div>
-        <div className={styles.badgeTitleWrapper}>
-          <div className={styles.badgeTitle}>
-            <Typography.H3 ellipsis={{ rows: 2, tooltip: { ...tooltipProps, title: customBadge.title } }}>
-              {customBadge.title}
-            </Typography.H3>
-            {customBadge.extra}
-          </div>
-          {customBadge.subtitle && (
-            <div className={styles.badgeSubtitle}>
-              <Typography.BodyS ellipsis={{ rows: 1, tooltip: { ...tooltipProps, title: customBadge.subtitle } }}>
-                {customBadge.subtitle}
-              </Typography.BodyS>
-            </div>
-          )}
-        </div>
-      </div>
+      <Badge {...customBadge} />
```

Comment:
Do we need the memo here?

Reply:
Yeah, memoization it's probably not necessary for all elements of this component since they are all very lightweight. I have removed them.